app/init.py

```diff
@@ -2,6 +2,14 @@
 import redis
 from urllib.parse import urlparse
 
+def waitForRDBLoad(conn):
```

Comment:
If you start the container without a flag to remove volumes, you will get an rdb that will be loaded on the second run. The code already has a key that is set at the end of the model storing and gear loading to skip reloading everything, so I just added a waiting loop because the ping from the client resulted in an exception

Reply:
can we run the server without persitency?